std/stdio.d

```diff
@@ -3258,14 +3258,20 @@ struct LockingTextReader
 }
 
 /**
- * Indicates whether $(D T) is a file handle of some kind.
+ * Indicates whether $(D T) is a file handle, i.e. the type
+ * is implicitly convertable to $(LREF File) or a pointer to a
+ * $(REF FILE, core,stdc,stdio).
+ *
+ * Returns:
+ *      `bool`
```

Comment:
What is our best practice for the return value?
Personally I think being a bit more explicit e.g. " `true` if it's a file handle, `false` otherwise" doesn't hurt.